<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="2">

  <testsuite name="exercises"
    errors="0" failures="1" hostname="" id="" package="exercises" skipped="1"
    tests="2" time="4.811" timestamp="Thu, 19 Apr 2018 12:11:46 GMT">
  
    <testcase name="Test 1" classname="exercises" time="4.811" assertions="1">  <failure message="Testing if the URL contains &#34;www.google.nl&#34;.">    at Object.Test 1 (/Users/yvarbergman/Projects/Module-Test-Automation/tests/exercises.js:6:20)
    at process._tickCallback (internal/process/next_tick.js:150:11)</failure>

    
    </testcase>
  

  

  
    
    <testcase
      name="Test 2" classname="exercises">
      <skipped />
    </testcase>
    
  
  </testsuite>
</testsuites>
